pom.xml

```diff
@@ -28,11 +28,12 @@
 
   <properties>
     <version.io.termd>1.0.0-20150807</version.io.termd>
+    <version.pnc.common>0.6-SNAPSHOT</version.pnc.common>
     <version.undertow-websockets-jsr>1.2.6.Final</version.undertow-websockets-jsr>
     <version.junit>4.11</version.junit>
     <version.org.slf4j>1.7.7</version.org.slf4j>
     <version.commons-cli>1.3</version.commons-cli>
-    <version.jackson>2.5.4</version.jackson>
+    <version.jackson>2.5.3</version.jackson>
 
```

Comment:
could not get it work, but managed now -> so moving back to 2.5.4 in the next commit